code/stable/ssp/SRS/SSP_SRS.tex

```diff
@@ -144,8 +144,14 @@ \subsection{Table of Symbols}
 \\
 ${{\mathbf{U}^{R}}_{t}}$ & Right Surface Hydrostatic Forces on Slices: per meter in the z-direction from water pressure acting into each slice from standing water on the slope surface, assuming the entire slice has the height of the right side of the slice & $\frac{\text{N}}{\text{m}}$
 \\
+$V$ & Volume: the amount of space that a substance or object occupies. & $\text{m}^{3}$
+\\
 $v$ & Local Index: used as a bound variable index in calculations & --
 \\
+${\mathbf{V}_{wat}}$ & Volumes of Water: amount of space occupied by water for each slice & $\text{m}^{3}$
+\\
+$W$ & Weight: The gravitational force acting on an object & N
```

Comment:
I thought for a second that having unbold W and bold W might be confusing, but the more I thought about it the more I'm okay with it.  unbold W on its own is not a component of bold W.  The components of bold W need an index to represent them, so there is no confusion between unbold W and bold W.